atoMEC/mathtools.py

```diff
@@ -1,9 +1,25 @@
-"""
-Low-level module containing various mathematical functions
+r"""
+Low-level module containing miscalleneous mathematical functions.
+
+Functions
+---------
+normalize_orbs: normalize KS orbitals within defined sphere
```

Comment:
You may want to consider using a [Python object reference](https://www.sphinx-doc.org/en/master/usage/restructuredtext/domains.html#cross-referencing-python-objects) if you want to list functions, classes etc explicitly in the modules description like so (no need to specify the `:py:` domain as it's the default domain in Sphinx)
```suggestion
:func:`normalize_orbs`: normalize KS orbitals within defined sphere
```
This generate a click able hyper link to the functions documentation:
![grafik](https://user-images.githubusercontent.com/28047702/124566457-58dd3800-de43-11eb-96bb-adbe5b9cb2bc.png)

Good examples for module docstrings can be found in scikit-learn, for example [here](https://raw.githubusercontent.com/scikit-learn/scikit-learn/main/doc/modules/calibration.rst).